Cargo.toml

```diff
@@ -11,7 +11,9 @@ categories = ["cryptography", "cryptography::cryptocurrencies"]
 [workspace]
 members = [
   "bin/hera",
+  "bin/reth",
   "crates/kona-providers",
+  "crates/hera-exex",
```

Comment:
WDYT about calling this crate `rollup` and inside, calling the `HeraExEx` the rollup `Driver`?

Then we can avoid pigeon-holing this crate to be specifically for an exex or even more specifically the `hera` binary.

Reply:
Otherwise really like the refactor to keep the binaries light.

Reply:
Nice, for sure